queries/dart.scm

```diff
@@ -16,3 +16,17 @@
   (set_or_map_literal)
   (map_pattern)
 ] @map
+
+;;!! class Foo {}
+;;!  ^^^^^^^^^^^^
+(
+  [
+    (class_definition
+      name: (_) @className
+    )
+    (class_definition
+
+      name: (_) @className
+    )
+  ] @class
```

Comment:
The domain of class name should be the whole class, so that you can say "class name" from anywhere in the class

Note you'll need to rerun update fixtures after this change

```suggestion
  ] @class @className.domain
```